4.7.1/build/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# escape=`
```

Comment:
Any reason you had to change the escape character? If possible, I'd like to be consistent across the build and runtime Dockerfiles

Reply:
I had to switch it at one point when I was using a multi-stage build.  I ended up in a situation where double escaping of `\` was needed which was pretty ugly.  I didn't end up going down that approach so I could revert this.  I will point out however that Docker points out the usefulness of using ` in Windows Dockerfiles to avoid issues - see https://docs.docker.com/engine/reference/builder/#escape.  I agree consistency across all Dockerfiles is best.  What is your opinion on switching it even though it is not strictly necessary at this point?